.github/workflows/check-release-target-tuples.yml

```
@@ -0,0 +1,41 @@
+# deny.tomlの`graph.targets`がbuild and deploy workflowのものと一致しているかを検査する。
+name: Check release target tuples
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
+      - .github/workflows/check-release-target-tuples.yml
+      - .github/workflows/build_and_deploy.yml
+      - deny.toml
+
+jobs:
+  check:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Install go-yq v4
+        run: |
+          mkdir -p ~/.local/bin
+          echo ~/.local/bin >> "$GITHUB_PATH"
+          tag=$(
+            gh release ls -R mikefarah/yq --json tagName,isPrerelease \
+              -q 'map(select(.isPrerelease | not).tagName | select(test("^v4\\.")))[0]'
+          )
+          gh release download -R mikefarah/yq "$tag" -p yq_linux_amd64 -O ~/.local/bin/yq
+          chmod u+x ~/.local/bin/yq
+          ~/.local/bin/yq -V
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

aptのが楽かも
https://mikefarah.gitbook.io/yq/v3.x#on-ubuntu-16.04-or-higher-from-debian-package

```suggestion
      - name: Install yq
        run: |
          add-apt-repository ppa:rmescandon/yq
          apt update
          apt install yq -y
```

---

sudoが必要なのはいいとして、なんか上手くいかない。
<https://github.com/VOICEVOX/voicevox_core/actions/runs/13743959596?pr=1012>

---

というかなんか日付が古そうだし、yqのreadmeでも紹介されていないっぽい…？

---

すみません、ちょっとよくわからなかったのでリバートしてGitHub Releasesのに戻しました。

---

動かなかったの悲しい。。すみませんでした。。 🙇